pkg/embeddedetcd/config.go

```diff
@@ -98,7 +98,7 @@ func NewConfig(o options.CompletedOptions, enableWatchCache bool) (*Config, erro
 		if err != nil {
 			return nil, err
 		}
-		cfg.ListenMetricsUrls = append(cfg.LPUrls, *u)
+		cfg.ListenMetricsUrls = append(cfg.LPUrls, *u) // TODO: What was the intent here?
```

Comment:
in which sense? Converting strings into URLs it seems.

Reply:
Your diff, I think? https://github.com/kcp-dev/kcp/commit/2b9193adc750b7005b3b5108d8be3cc09365fae5#diff-aece0d70693f229fcbf4f1ca12cc1beec3066aa4059b0a054b924d790b9c8b23L107-L109

Reply:
Won't during the loop the result of the append() constantly override `cfg.ListenMetricsUrls`? So only the last iteration of the loop will actually have any effect.

Reply:
https://github.com/kcp-dev/kcp/pull/1907#discussion_r968618435